app/konnect/runtime-manager/configuration/vaults/how-to.md

```diff
@@ -20,6 +20,10 @@ Open the **Vaults** page in {{site.konnect_short_name}}:
 Set up a new vault. For this example, we're going to use the environment variable vault. 
 1. Click **Add vault**.
 1. Choose a vault type. 
+1. Enter the configuration settings for your vault. For more information about how to configure settings, see the following {{site.base_gateway}} documation:
+    * [AWS vault configuration options](/gateway/latest/kong-enterprise/secrets-management/backends/aws-sm/#vault-configuration-options)
+    * [Google Cloud vault configuration options](/gateway/latest/kong-enterprise/secrets-management/backends/gcp-sm/#vault-entity-configuration-options)
+    * [HashiCorp vault configuration options](/gateway/latest/kong-enterprise/secrets-management/backends/hashicorp-vault/#vault-configuration-options)
```

Comment:
❗ Reviewers: Do you like the links to Gateway docs? I feel like the UI isn't helpful in telling you what you need to put there or what they do, so I felt like we needed to include something. This is kind of an "MVP" solution, I think maybe the Konnect docs need their own table? Or maybe we just continue linking to Gateway to point to how the two interact?

Reply:
The links work for me. I don't think we should worry about this too much on our end; if anything, we should be pointing out areas for UI improvements.